libcxx/src/iostream.cpp

```diff
@@ -38,13 +38,20 @@ union stream_data {
 #define CHAR_MANGLING_wchar_t "_W"
 #define CHAR_MANGLING(CharT) CHAR_MANGLING_##CharT
 
+#ifdef _LIBCPP_COMPILER_CLANG_BASED
+#  define STRING_DATA_CONSTINIT constinit
```

Comment:
can't figure out how to make GCC happy,
all the time:  error: ‘constinit’ variable ‘std::__1::cin’ does not have a constant initializer

Reply:
Hmm. I'll try to figure something out tomorrow for this. Either a bug against one of the compilers or some code changes should happen here. I guess a CWG issue would also be possible.

Reply:
Can we land this one in meantime?

Reply:
I'm fine with that.